scripts/install-dev-tools.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Personally I think we should not have this script and the debug image also doing it's own thing.
We should do either:

1. Update the debug image to include everything missing: https://github.com/fluent/fluent-bit/blob/afb2ba2bbff09557f5496c1b198b056112b84449/dockerfiles/Dockerfile#L207-L227
2. Update the debug image to call this script.

I would go with 1 personally - this script is not useful to anyone outside of a Debian ecosystem so would need updating to handle Yum at the very least when it detects that platform. 

You are also going to have to cope with different versions of the OS (e.g. Buster, Bullseye, CentOS 7, 8, 9) and different config or dependencies for each so it gets messy fast. Currently it only really works on the same config as the debug image - it needs `bullseye-backports` set up for example.

My suggestion is to update the Dockerfile so it adds any missing components and remove this script.

Reply:
I agree with you, I think first option is better than the other one.